### ISSUE TITLE
Ntfy notification type - button in notification does nothing

### ISSUE THREAD

**Issue by k-matti** [louislam/uptime-kuma#4742]
### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/uptime-kuma/security/policy)

### Description

There is a button in notification "OPEN EMBY" that points to "https://"
![obraz](https://github.com/louislam/uptime-kuma/assets/66017397/c84aba33-1161-46ab-b3f8-c3cee94e9ab9)

What is a purpose of this button?

### 👟 Reproduction steps

1. Set docker container monitor type
2. Add NTFY notification
3. Stop container

### 👀 Expected behavior

Button should not be present at all or should be present when container back to normal with url to the service

### 😓 Actual Behavior

button does nothing

### 🐻 Uptime-Kuma Version

1.23.13

### 💻 Operating System and Arch

Unraid

### 🌐 Browser

Firefox

### 🖥️ Deployment Environment

- Runtime: Docker

**Comment by CommanderStorm** [comment]
https://github.com/louislam/uptime-kuma/pull/4411 has been merged into the v2.0-milestone adressing this.
You can see #4500 for the steps that are left before we can publish the first beta.
